frontend/src/components/modules/GameInterface/GameInterface.tsx

```diff
@@ -701,11 +710,21 @@ const GameInterface: React.FC = () => {
                       >
                         {t("game_interface.reset")}
                       </Button>
-                    </Grid>
-                    <br />
-                    <br />
+                    ) : (
+                      // Reset roles button
+                      // Only shown for the tournament creator before the match starts
+                      <Button
+                        variant="contained"
+                        onClick={async () => {
+                          await handleResetRoles();
+                        }}
+                      >
+                        {t("game_interface.reset_roles")}
+                      </Button>
+                    )}
```

Comment:
toi ? ja : tapa ei oikee toimi täs koska nyt tää buttoni tulee näkyy sit ku matsi päättyy koska sillon endtimestamp ei oo enää undef nii se valitsee : polun if that makes sensee